src/components/FeaturedContainer/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import CategoryCard from "../CategoryCard";
+import "./featured-container.css";
+const FeaturedContainer = (props) => {
+  const { featuredContainerTitle } = props;
+  return (
+    <div className="margin-top-60">
+      <h3 className="featured-container-header">{featuredContainerTitle}</h3>
+      <div className="featured-container">
+        <CategoryCard categoryTitle={"Hiking and Trekking"} />
+        <CategoryCard categoryTitle={"Wildlife Watching"} />
+        <CategoryCard categoryTitle={"Swimming"} />
+        <CategoryCard categoryTitle={"Horse Riding"} />
+        <CategoryCard categoryTitle={"Rock Climbing"} />
+        <CategoryCard categoryTitle={"Skiing and Snowboarding"} />
+        <CategoryCard categoryTitle={"Fishing"} />
+        <CategoryCard categoryTitle={"T-shirts and Trousers"} />
```

Comment:
Suggestion: make a list in which add all the text, then do .map to it this will help in scalability

Reply:
fixed.